Jenkinsfile

```diff
@@ -56,7 +56,7 @@ node("postgresql-9.6") {
         runPactTests(govuk, "frontend", FRONTEND_BRANCH)
         runPactTests(govuk, "account-api", ACCOUNT_API_BRANCH, [ resetDatabase: true ])
         runPactTests(govuk, "link-checker-api", LINK_CHECKER_API_BRANCH, [ resetDatabase: true ])
-        runPactTests(govuk, "imminence", IMMINENCE_BRANCH, [ resetDatabase: true ])
+        runPactTests(govuk, "imminence", IMMINENCE_BRANCH, [ resetDatabase: true, createIndexes: true ])
```

Comment:
These changes can be rebased into 2c47625da56bd00f4a64453db732f93fd3b61f94, since the original Jenkinsfile changes won't have worked in isolation and therefore need this change too.

We should also explain in the commit why this is necessary.

Reply:
👍